src/js/components/Inspector.jsx

```diff
@@ -12,7 +12,7 @@ var Inspector = React.createClass({
     var props = this.props,
         primitive = lookup(props.id),
         from = lookup(primitive.from),
-        ctor = primitive.getName(),
+        ctor = primitive.constructor.name,
```

Comment:
Wouldn't this need to be primitive.prototype.constructor.name...? I'm not sure I understand the nuance of what this is doing.